src/transport/MessageCounter.h

```diff
@@ -106,8 +108,14 @@ class GlobalEncryptedMessageCounter : public MessageCounter
 class LocalSessionMessageCounter : public MessageCounter
 {
 public:
-    static constexpr uint32_t kInitialValue = 1;
-    LocalSessionMessageCounter() : value(kInitialValue) {}
+    static constexpr uint32_t kInitialValue = 1;    ///< Used for initializing peer counter
+
+    /**
+     * Initialize a local message counter with random value between [0, 2^28-1].
+     */
+    LocalSessionMessageCounter() {
+        value = Crypto::GetRandU32() & 0x0FFFFFFF;
```

Comment:
Maybe introduce and use here new constant:
`constexpr suint32_t k28BitMask = 0x0FFFFFFF;`

Reply:
I applied the suggestion, but in general constexpr increases flash size relative to a #define macro or enum.